uint16[] drive_ids
---
bool[] drives_success